packages/gp2-components/src/molecules/FilterPill.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  borderWidth,
+  crossSmallIcon,
+  pixels,
+} from '@asap-hub/react-components';
+import { css } from '@emotion/react';
+import { ReactNode } from 'react';
+import colors from '../templates/colors';
+
+const { rem } = pixels;
+
+const pillStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  width: 'fit-content',
+  height: rem(22),
+  padding: `${rem(8)} ${rem(12)}`,
+  gap: rem(8),
+  borderStyle: 'solid',
+  borderWidth: `${borderWidth}px`,
+  borderRadius: rem(24),
+  borderColor: colors.neutral500.rgb,
+  color: colors.neutral1000.rgb,
+});
+
+const iconStyles = css({
+  display: 'flex',
```

Comment:
this is just a button with icon in, I dont think we need any flex styling in, It would also be nice to add the `cursor: 'pointer'` we do for Button

Reply:
Without the flex it will not center in it's div, but I've tested a lil bit more and it was possible to remove the other 2 properties and added the cursor : 'pointer'